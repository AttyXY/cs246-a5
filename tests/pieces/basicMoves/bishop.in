setup
+ k a1
+ K h8
+ p b2
+ B a8
done
game human human
move a8 h1
move a1 a2
move h1 e4
move a2 a3
move e4 b1 
move a3 a4
move b1 h7 
move a4 a3
move h7 b1
quit
quit
