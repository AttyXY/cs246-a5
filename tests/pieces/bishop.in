game human human 
move b2 b4
move b7 b6
move c1 b2
move c8 b7
move b2 g7
move b7 e4
quit
quit
