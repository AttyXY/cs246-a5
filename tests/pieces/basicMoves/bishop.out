WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  -p------
1  k-------
   abcdefgh

8  B------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  -p------
1  k-------
   abcdefgh

STARTING GAME!

8  B------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  -p------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  -p------
1  k------B
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  kp------
1  -------B
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  ----B---
3  --------
2  kp------
1  --------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  ----B---
3  k-------
2  -p------
1  --------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  k-------
2  -p------
1  -B------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  k-------
3  --------
2  -p------
1  -B------
   abcdefgh

8  -------K
7  -------B
6  --------
5  --------
4  k-------
3  --------
2  -p------
1  --------
   abcdefgh

8  -------K
7  -------B
6  --------
5  --------
4  --------
3  k-------
2  -p------
1  --------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  k-------
2  -p------
1  -B------
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
